### PR TITLE
Make Westmint Support Bridged Assets

### PR DESCRIPTION
**Pull request by joepetrowski**
Support for `Foreign` assets

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2187130

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2253708

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2256815

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2257035

File: parachains/runtimes/assets/westmint/src/xcm_config.rs [hunk 1]
```diff
@@ -251,3 +261,44 @@ impl cumulus_pallet_xcm::Config for Runtime {
...
+
+	#[cfg(feature = "runtime-benchmarks")]
+	fn successful_origin(a: &MultiLocation) -> RuntimeOrigin {
+		pallet_xcm::Origin::Xcm(a.clone()).into()
+	}
+}
+
+/// Simple conversion of `u32` into an `AssetId` for use in benchmarking.
+pub struct XcmBenchmarkHelper;
+use pallet_assets::BenchmarkHelper;
```
Comment (bkontur) [inline parachains/runtimes/assets/westmint/src/xcm_config.rs:298]:
```suggestion
#[cfg(feature = "runtime-benchmarks")]
use pallet_assets::BenchmarkHelper;
```

File: parachains/runtimes/assets/westmint/src/lib.rs [hunk 2]
```diff
@@ -238,6 +269,8 @@ impl pallet_assets::Config for Runtime {
 	type Extra = ();
 	type WeightInfo = weights::pallet_assets::WeightInfo<Runtime>;
 	type AssetAccountDeposit = AssetAccountDeposit;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Helper = XcmBenchmarkHelper;
```
Comment (bkontur) [inline parachains/runtimes/assets/westmint/src/lib.rs:273]: @joepetrowski 
please, see: https://github.com/paritytech/cumulus/commit/25c2fc5bbb8a99963928aa20a7029d79604ef453#r95961973

and other instance has only `type Helper = ();` ?

Comment (joepetrowski) [reply parachains/runtimes/assets/westmint/src/lib.rs]: `()` assumes that you can cast `u32` into the `AssetId`. We need to mock something for the case where we have `MultiLocation` as an `AssetId`.

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2257652

**Pull request closed without merging**